code/tests/adb_test_AdobeEdge.brs

```diff
@@ -86,3 +86,13 @@ sub TestCase_AdobeEdge_adb_StateManager_edgeDomain()
     UTF_assertEqual(stateManager.getConfigId(), "testConfigId")
     UTF_assertEqual(stateManager.getEdgeDomain(), "abx")
 end sub
+
+' target: _adb_isEmptyOrInvalidString()
+' @Test
+sub TestCase_AdobeEdge_adb_isEmptyOrInvalidString()
+    UTF_assertTrue(_adb_isEmptyOrInvalidString(invalid))
+    UTF_assertTrue(_adb_isEmptyOrInvalidString(""))
+    UTF_assertFalse(_adb_isEmptyOrInvalidString("test"))
+    UTF_assertTrue(_adb_isEmptyOrInvalidString(123))
```

Comment:
@addb  I update the function name to `_adb_isEmptyOrInvalidString` to make it more clearer. Let me know if you have any thoughts about the new name.

Reply:
looks good, nit comment: `isInvalidOrEmptyString()`